### ISSUE TITLE
Is translations possible?

### ISSUE THREAD

**Issue by 1upus** [Rinnegatamante/HCL-Vita#4]
Is it possible move all hardcoded text strings to single file and use it for translations?. Thx.

**Comment by Rinnegatamante** [comment]
They're already in separated files (en.dat , jp.dat)

**Comment by 1upus** [comment]
Is it any aditor for text/font? As I understand, all fonts packed into bmp.qda.

**Comment by Rinnegatamante** [comment]
There's only a font packed in bmp.qda related to standard game text.
Btw don't think there's an editor out there.
The font however can be easily extracted, bmp.qda is just an archive of bmp images.

**Comment by 1upus** [comment]
I'm found a tool to export/imporf gfx from bmp.qda
https://github.com/JeffRuLz/HCL-Graphics-Manager/releases
But cant understand text encoding in en.dat. Cab it be packed of encrypted?
Also splashscreen on game start have some text need to be translated. As I understand it packer to eboot.bin?

**Comment by Rinnegatamante** [comment]
If you mean livearea contents, they're in build folder and are created during VPK creation.

**Comment by 1upus** [comment]
No, I told about splashscreen before “new game/continue”. It uses ingame font and says about vita version.

**Comment by Rinnegatamante** [comment]
It's in the code like the main menu too: https://github.com/Rinnegatamante/HCL-Vita/blob/master/source/titlescreen.c
